packages/lit-dev-content/site/blog/2021-09-22-lit-2-release-livestream.md

```diff
@@ -0,0 +1,15 @@
+---
+tags: blogPosts
+layout: blog-post.html
+title: "Watch the Lit 2 release livestream"
+summary: "What's new in Lit 2, a big-picture view of Lit usage at Google, and a community panel discussion."
+date: 2021-09-22
+---
+
+<div class="preamble">
+
+If weren't able to join us for the Lit 2 the release livestream, you can watch the recording on YouTube.
```

Comment:
Nit: Add `you`: "If **you** weren't"

Remove `the` from "Lit 2 ~~the~~ release"

Reply:
Fixed, thanks!